public/index.html

```diff
@@ -3,6 +3,17 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="./images/general/newlogo.png" />
+    <link
```

Comment:
isn't there a way of bundling this css with the app?

https://react-slick.neostack.com/docs/get-started

seems you can 

```bash
npm install react-slick slick-carousel
```

and the import the css in your global css file

```css
import "~slick-carousel/slick/slick.css"; 
import "~slick-carousel/slick/slick-theme.css";
```

you shouldn't need to include the css as links

Reply:
thank you for the observation! done